### ISSUE TITLE
Line edits in docs

### ISSUE THREAD

**Issue by elmps2018** [odk-x/tool-suite-X#207]
This issue is an ongoing, open issue for any line edits to the docs. If you are making minor fixes that are not related to another issue, you can refer to this one!

**Comment by ChitvanRamani22** [comment]
@elmps2018 can i work on this

**Comment by elmps2018** [comment]
@ChitvanRamani22 this is an ongoing, open issue, which anyone can (non-exclusively) work on if fixing any minor issues in docs (if you find a typo, bad link, etc.).

**Comment by SaySayo** [comment]
can i work on this

**Comment by wbrunette** [comment]
@SaySayo this is an ongoing, open issue, which anyone can (non-exclusively) work on if fixing any minor issues in docs (if you find a typo, bad link, etc.).

**Comment by iamhks** [comment]
@elmps2018 I have made a pull request please review here https://github.com/odk-x/docs/pull/72

**Comment by wbrunette** [comment]
@iamhks thanks for fixing the typo. In the future can you please update the pull request template to point to this issue so when this comes up "addresses odk-x/tool-suite-X#" you add the number, don't leave it blank. This will automatically link it to this issue.

**Comment by iamhks** [comment]
@wbrunette Yeah sure, I'll take care of that from next time. Thanks.

**Comment by pawanharariya** [comment]
@elmps2018 Some things are missing on this page https://docs.odk-x.org/docs-tech-guide/#keep-improving . In the `setup collaboration accounts` section it says three platforms but only mentions github and odk-x forum. I guess the third one is slack. Same issue is seen in some other places in this section. How should I correct this?

**Comment by elmps2018** [comment]
@pawanharariya you are right there should be two platforms with github and ODK-X forum. You can make line edits in the docs following the docs contribution directions that you found this error in :)

**Comment by OyinOlamide** [comment]
Hi @EstherPhebe ,Can I please be assigned to this issue? And how do I start, please?

**Comment by greatestofal** [comment]
Hello @elmps2018, can I please work on this? What do I need to get started? Thanks

**Comment by elmps2018** [comment]
Hi @OyinOlamide and @greatestofal this is an ongoing, open issue, which anyone can (non-exclusively) work on if fixing any minor issues in docs (if you find a typo, bad link, etc.). If your pull request is focused on fixing an error please just reference this issue. Thanks!

**Comment by JudeUk** [comment]
Hello @elmps2018 i need your help please, following this guide (https://docs.odk-x.org/contributing/), i decided to give Issue 207 — Line Edits a try as a first contribution. I have set up my docs contributing environment locally and even went on to make some modifications on a file. My problem is at the point of serving the site locally, to see my edit on a browser, i get this error when i go to the URL on my browser.

Please see attached screenshots of my browser and terminal thank you.
<img width="1440" alt="Screenshot 2021-10-16 at 16 47 20" src="https://user-images.githubusercontent.com/31444432/137692245-1083ed31-74b1-4c40-8a6a-63e227b5e7f9.png">
<img width="1440" alt="Screenshot 2021-10-16 at 16 47 37" src="https://user-images.githubusercontent.com/31444432/137692267-5916e977-500a-40fe-a47c-e6d4c099e181.png">

**Comment by elmps2018** [comment]
Hi @JudeUk! Thanks for trying to work on the docs! So it looks like the build and checks are not working either (earlier errors in your screenshot). This suggests something has gone wrong with the initial setup. Did you get any error messages or issues when following the setup directions?

**Comment by JudeUk** [comment]
Hi @elmps2018 you are right, the checks wasn't working. I went through the instructions again and was able to run the docs app locally. Thanks once again

**Comment by JudeUk** [comment]
Hello @elmps2018 i just made a Pr kindly review. thank you

**Comment by permission-error1** [comment]
@wbrunette  @elmps2018 I need your reviews here https://github.com/odk-x/docs/pull/164

**Comment by permission-error1** [comment]
@elmps2018 could you review this one too https://github.com/odk-x/docs/pull/165

**Comment by permission-error1** [comment]
@elmps2018 could you review here please https://github.com/odk-x/docs/pull/166

**Comment by permission-error1** [comment]
@elmps2018  could you review this https://github.com/odk-x/docs/pull/168

**Comment by MariaAngella** [comment]
Hello @wbrunette @elmps2018, 

I have Added Introduction video on the README.md file. [Adding youtube video intro #433](https://github.com/odk-x/tool-suite-X/pull/433)
Please advise if this can help improve on the instructions.


Thank you and Regards,
Maria Angella

**Comment by wbrunette** [comment]
@Chinex-Boroja thank you for doing this investigation. Good Job! Because of your description, I figured out the issue quickly and it should no longer an issue. Therefore, I am going to delete your comment (though it is included in this message for history sake) so people are not confused. Include you figuring this out in your Outreachy application!

> Hello everyone, a few tips to note in order to avoid `block-unregistered-user` error message, which seems to be causing the build failures. This is for those working on the docs
> 
>     * Sign up for [Circle CI](https://circleci.com/signup/) (Preferably use Github to authorize your signup). Make sure you are logged in
> 
>     * Delete the previous PR that its build tests were failing
> 
>     * Delete the feature branch you made the PR from on GitHub
> 
>     * Push another commit to Github
> 
>     * Lastly, create another PR (Your build should pass)

**Comment by siva224513** [comment]
I would like to working on this issue

**Comment by wbrunette** [comment]
@siva224513  this is an ongoing, open issue, which anyone can (non-exclusively) work on if fixing any minor issues in docs (if you find a typo, bad link, etc.).

**Comment by Dev-JoyA** [comment]
@wbrunette @elmps2018 These are the issues I ran into while trying to set -up **application Designer** and this is how i resolved them.
These are the issues i encountered when setting up ODK-X Application Designer for macOs Version 12.7.1 . 

when I ran this command  ` ./sdkmanager --list` in  step 7 in the installation page: "https://docs.odk-x.org/app-designer-prereqs/#nodejs"

I got this error:

```sh
Exception in thread "main" java.lang.UnsupportedClassVersionError: com/android/sdklib/tool/sdkmanager/SdkManagerCli has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:473)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
```

Here's how i resolved the issue:

```sh
SOLUTION
```

1. you need to upgrade Java to version 17 or older
2. In the Andriod SDK application , 
- Go to tools
- Go to SDK manager
- Then Andriod SDK
- Click on SDK Tools
 Make sure Everything in the Andriod command line tools is checked. (you can think the box show package details to see the drop down and make sure everything is checked)
- Then click apply
Some Installation will run , then click finish and then Ok.

Continue with step the Note after Step 7 in "https://docs.odk-x.org/app-designer-prereqs/#nodejs"
And Everything will work accordingly.
After all this you can go to your terminal cd in to the right path and run any sdkmanager command .

**Comment by Dev-JoyA** [comment]
@wbrunette @elmps2018 I would like to work on ODK-X documentation as my contribution and make it more friendly and detailed for new users or intern in this case

**Comment by wbrunette** [comment]
@Dev-JoyA as I commented 2 days ago, this is an ongoing, open issue, which anyone can (non-exclusively) work on if fixing any minor issues in docs (if you find a typo, bad link, etc.).

**Comment by TunrayoIlawole** [comment]
Hi, @wbrunette . I found this issue on the https://docs.odk-x.org/tables-sample-app-tea-houses/ page of the documentation (see blue highlighted lines).
<img width="1430" alt="Screenshot 2024-10-02 at 13 19 04" src="https://github.com/user-attachments/assets/cf104b89-84f1-4250-b50e-9cbf629244a4">


 I believe there should be an image showing the combination of the detail and list view options and the image directive has not been properly added.
Kindly help advise on the correct image to use (i think it should be this image - "tables-sample-tea-detail-sublist.png") and let me know if i can go ahead and work on this. Thank you
